### ISSUE TITLE
No reflections in 2022.1.17f1

### ISSUE THREAD

**Issue by ShockWave2048** [ColinLeung-NiloCat/UnityURP-MobileScreenSpacePlanarReflection#9]
Hi! Cant get reflections in 2022.1.17f1.
Some errors here.
Can you help please?
![Screenshot_7](https://user-images.githubusercontent.com/800625/192131005-9888fd67-4bd2-4e47-b914-8b64a1a76455.png)

**Comment by Gu-yusi** [comment]
2022.3.3 too :(

**Comment by androacks22** [comment]
Have they not solved the problem yet? I really need it for my video game :(

**Comment by Gu-yusi** [comment]
> Have they not solved the problem yet? I really need it for my video game :(

you may need this：）
![屏幕截图 2023-10-27 125431](https://github.com/ColinLeung-NiloCat/UnityURP-MobileScreenSpacePlanarReflection/assets/97105581/5822b8ac-4c93-4732-be29-86a023e74880)

**Comment by androacks22** [comment]
> esto

Honestly, I don't quite understand much, I'm sorry for my clumsiness :(. Should I change something in the code or what? Can you help me?

**Comment by Gu-yusi** [comment]
> > esto
> 
> Honestly, I don't quite understand much, I'm sorry for my clumsiness :(. Should I change something in the code or what? Can you help me?

Yes, you need to modify the MobileSSPRRendererFeature.cs script, the way of obtaining _CameraOpaqueTexture and _CameraDepthTexture has changed.
![屏幕截图 2023-10-27 141551](https://github.com/ColinLeung-NiloCat/UnityURP-MobileScreenSpacePlanarReflection/assets/97105581/d8fe408d-47b5-4bb5-a5bd-3971b534844d)
![image](https://github.com/ColinLeung-NiloCat/UnityURP-MobileScreenSpacePlanarReflection/assets/97105581/2c868b3e-e16b-433b-8da0-4307da8963b7)
![Uploading image.png…]()

**Comment by Gu-yusi** [comment]
> > esto
> 
> Honestly, I don't quite understand much, I'm sorry for my clumsiness :(. Should I change something in the code or what? Can you help me?

Yes, you need to modify the MobileSSPRRendererFeature.cs script, the way of obtaining _CameraOpaqueTexture and _CameraDepthTexture has changed.
![屏幕截图 2023-10-27 141551](https://github.com/ColinLeung-NiloCat/UnityURP-MobileScreenSpacePlanarReflection/assets/97105581/aa0c60c4-f329-447c-9b53-c9d181936a1c)
![屏幕截图 2023-10-27 141702](https://github.com/ColinLeung-NiloCat/UnityURP-MobileScreenSpacePlanarReflection/assets/97105581/d1564bc3-aa24-411e-954f-f9b513a3e5bf)
![屏幕截图 2023-10-27 141734](https://github.com/ColinLeung-NiloCat/UnityURP-MobileScreenSpacePlanarReflection/assets/97105581/e9f03616-8ba8-48b0-a2cb-a6c8dc4106c9)

**Comment by androacks22** [comment]
> 
> Sí, necesita modificar el script MobileSSPRRendererFeature.cs, la forma de obtener _CameraOpaqueTexture y _CameraDepthTexture ha cambiado. ![屏幕截图 2023-10-27 141551](https://user-images.githubusercontent.com/97105581/278553478-aa0c60c4-f329-447c-9b53-c9d181936a1c.png) ![屏幕截图 2023-10-27 141702](https://user-images.githubusercontent.com/97105581/278553582-d1564bc3-aa24-411e-954f-f9b513a3e5bf.png) ![屏幕截图 2023-10-27 141734](https://user-images.githubusercontent.com/97105581/278553602-e9f03616-8ba8-48b0-a2cb-a6c8dc4106c9.png)

The warnings are gone, but now it's doing very strange things.😓
(It's a test project; my game is in another project. So, excuse the clutter in the assets)
![Captura de pantalla 2023-10-27 022218](https://github.com/ColinLeung-NiloCat/UnityURP-MobileScreenSpacePlanarReflection/assets/68607621/c3834c99-58e7-4756-997f-49e5965bfd4d)

**Comment by Gu-yusi** [comment]
@androacks22 You should set the graphics API in Player to Direct3D12

**Comment by androacks22** [comment]
> @androacks22 You should set the graphics API in Player to Direct3D12

thank you very much for the help, it worked.
